service-base/src/main/java/org/eclipse/hono/service/AbstractServiceBase.java

```diff
@@ -399,7 +403,7 @@ protected final void addTlsKeyCertOptions(final NetServerOptions serverOptions)
             final boolean isOpenSslAvailable = OpenSsl.isAvailable();
             final boolean supportsKeyManagerFactory =  OpenSsl.supportsKeyManagerFactory();
             final boolean useOpenSsl =
-                    getConfig().isNativeTlsRequired() || (isOpenSslAvailable && supportsKeyManagerFactory);
+                    getConfig().isNativeTlsRequired() || isOpenSslAvailable && supportsKeyManagerFactory;
```

Comment:
is there a reason why you removed the brackets?

Reply:
Eclipse did that for me, as they are unnecessary.

Reply:
But they improve readability a lot, don't you think? I have to admit that I would have to look up operator preference in order to predict the expression's value ...

Reply:
For me more brackets are more confusing … but I can restore them and disable the cleanup action.

Reply:
I would appreciate it :+1:

Reply:
Restored by 85df64f15fd7b5ebea012b23b011140324d88e67